plugins/vpp/l3plugin/descriptor/vrrp.go

```diff
@@ -0,0 +1,255 @@
+//  Copyright (c) 2020 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package descriptor
+
+import (
+	"net"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/pkg/errors"
+	"go.ligato.io/cn-infra/v2/logging"
+	kvs "go.ligato.io/vpp-agent/v3/plugins/kvscheduler/api"
+	ifdescriptor "go.ligato.io/vpp-agent/v3/plugins/vpp/ifplugin/descriptor"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/l3plugin/descriptor/adapter"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/l3plugin/vppcalls"
+	interfaces "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/interfaces"
+	l3 "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3"
+)
+
+const (
+	// VrrpDescriptorName is the name of the descriptor.
+	VrrpDescriptorName = "vrrp"
+
+	// Dependency labels.
+	vrrpEntryInterfaceDep    = "interface-exists"
+	vrrpDescriptorLoggerName = "vrrp-descriptor"
+
+	maxUint8  = 255
+	maxUint16 = 65535
```

Comment:
These are defined in the [`math`](https://golang.org/pkg/math/#pkg-constants) package already.